client/src/js/services/NotifyService.js

```diff
@@ -13,7 +13,7 @@ function NotifyService($translate) {
   const service = this;
 
   // default time to live of 3 seconds
```

Comment:
You might want to update this comment as well 😄

Reply:
Thanks. Just pushed the fix.